docs/blog/pydantic-2-alpha.md

```diff
@@ -0,0 +1,57 @@
+# Pydantic 2 Alpha Release
+
+<aside class="blog" markdown>
+![Samuel Colvin](/img/samuelcolvin.jpg)
+<div markdown>
+  **Samuel Colvin** &bull;&nbsp;
```

Comment:
@tpdorsey I think we should make you the author here, happy to add me as well if you'd like.